docs/source/deployorderer/ordererchecklist.md

```diff
@@ -142,7 +142,8 @@ In general, these four parameters would only need to be configured if you want t
 BootstrapMethod: file
 ```
 
-* **`BootstrapMethod`**: If you plan to create this node on a network that is not using a system channel, override this value to `none` and then ensure that [`ChannelParticipation.Enabled`](#channelparticipation) is set to `true`, otherwise you will get an error when you attempt to start the node. If you are creating a node to be joined to a system channel, unless you plan to use a file type other than “file”, this value should be left as is.
+* **`BootstrapMethod`**: If you plan to create this node on a network that is not using a system channel, override this value to `none` and then ensure that [`ChannelParticipation.Enabled`](#channelparticipation) is set to `true`, otherwise you will get an error when you attempt to start the node. 
```

Comment:
You can entirely remove wording like "If you plan to create this node on a network that is not using a system channel". Just say that it must be set to 'none' because system channel is no longer supported, like you do in BoostrapFile below.